docs/windows.md

```diff
@@ -0,0 +1,29 @@
+# Windows support
+
+Windows support is under evaluation. This document describes issues which should be resolved before claiming full support for windows.
```

Comment:
What do you mean by `under evaluation`?

Reply:
That we are validating how it works, and we are going to make it working, but we are not going to do any claims in nearest future :sweat_smile:

Reply:
I add that this is experimental, but left also that it is under evalutation. I think it's important information that it is not clear yet how much we do support windows or can support it with minimal amount of work